### PR TITLE
fix(core): add date picker hover tooltip

### PR DESCRIPTION
**Pull request by ASokolov22**
#### Please provide a link to the associated issue.
Fixes https://github.com/SAP/fundamental-ngx/issues/5080

#### Please provide a brief summary of this pull request.
Fixes issue #4 about: add a mouse hover tooltip to date picker buttons

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/calendar/calendar-header/calendar-header.component.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
                 [attr.aria-disabled]="false"
                 type="button"
                 (click)="previousClicked.emit()"
+                [title]="previousAriaLabel"
```
Comment (Betrozov) [inline libs/core/src/lib/calendar/calendar-header/calendar-header.component.html:20]:
```suggestion
                [attr.title]="previousAriaLabel"
```

File: libs/core/src/lib/calendar/calendar-header/calendar-header.component.html [hunk 2]
```diff
@@ -72,6 +73,7 @@
                 [attr.aria-disabled]="false"
                 (click)="nextClicked.emit()"
                 type="button"
+                [title]="nextAriaLabel"
```
Comment (Betrozov) [inline libs/core/src/lib/calendar/calendar-header/calendar-header.component.html:76]:
```suggestion
                [attr.title]="nextAriaLabel"
```

File: libs/core/src/lib/date-picker/date-picker.component.html [hunk 3]
```diff
@@ -36,6 +36,7 @@
                     [class.is-expanded]="isOpen"
                     [attr.aria-expanded]="isOpen"
                     (click)="toggleCalendar()"
+                    [title]="displayCalendarToggleLabel"
```
Comment (Betrozov) [inline libs/core/src/lib/date-picker/date-picker.component.html:39]:
```suggestion
                    [attr.title]="displayCalendarToggleLabel"
```

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Comment by Betrozov** [comment]
@ASokolov22 your PR contains changes only to core not to platform
please change your PR title and commit message

File: libs/core/src/lib/date-picker/date-picker.component.html (hunk 3 above)
Comment (ASokolov22) [reply libs/core/src/lib/date-picker/date-picker.component.html]: Done

**Review by Betrozov — APPROVED** [review]

**Pull request merged**